src/overlays/actors/ovl_Obj_Tokei_Turret/z_obj_tokei_turret.c

```diff
@@ -5,6 +5,12 @@
  */
 
 #include "z_obj_tokei_turret.h"
+#include "objects/object_tokei_turret/object_tokei_turret.h"
+
+typedef enum {
+    /* 0 */ DISPLAY_TURRET_BASE,
+    /* 1 */ DISPLAY_TURRET_TOP,
+} TokeiDisplayState;
```

Comment:
This should also change since it's used with the macro
```suggestion
    /* 0 */ TURRET_TIER_BASE,
    /* 1 */ TURRET_TIER_TOP,
} TokeiTierType;
```